src/Tests/RegistrationTest.php

```diff
@@ -25,19 +25,133 @@ class RegistrationTest extends WebTestBase {
   public static $modules = ['decoupled_auth', 'user', 'system'];
 
   /**
+   * The user settings config.
+   *
+   * @var \Drupal\Core\Config\Config
+   */
+  protected $user_config;
+
+  /**
+   * The acquisition settings config.
+   *
+   * @var \Drupal\Core\Config\Config
+   */
+  protected $acquisition_config;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->user_config = $this->config('user.settings');
+    $this->acquisition_config = $this->config('decoupled_auth.settings');
+  }
+
+  /**
+   * Don't require email verification and allow registration by site visitors
+   * without administrator approval.
+   */
+  protected function allowUserRegistration() {
```

Comment:
I was imagining that this chunk of code would go in setUp() as you call it for every test, right?